inc/TRestGeant4ParticleCollection.h

```diff
@@ -41,9 +40,9 @@ class TRestGeant4ParticleCollection : public TObject {
     virtual TRestGeant4Particle GetParticle(int i) { return fParticles[i]; }
 
     virtual void RemoveParticles() { fParticles.clear(); }
-    virtual void AddParticle(TRestGeant4Particle ptcle) { fParticles.push_back(ptcle); }
+    virtual void AddParticle(TRestGeant4Particle particle) { fParticles.push_back(particle); }
```

Comment:
Can you pass TRestGeant4Particle as reference? AddParticle(TRestGeant4Particle &particle)
Also, it iw possible to replace push_back by emplace_back? You can also do "fParticles.push_back(std::move(particle) );"